icssc/AntAlmanac
# Zotistics graph pops up on hover

## Summary
- The Zotistics graph now pops up when the user hovers over the Zotistics course info button. The popup closes when the mouse moves away from the popup or button.
- The `Popover` component has some weird interactions with `OnMouseLeave` and `OnMouseEnter` events. When the mouse leaves the area, sometimes the popup doesn't close. To avoid this, we can use the `Popper` component, which is basically a lower-level `Popover`.
- Some minor styling adjustments were made. The popup is a bit smaller so that it doesn't take much space, and so that the "View on Zotistics" link doesn't get cut off.

Demo:

https://github.com/icssc/AntAlmanac/assets/78244965/3235ea48-e42a-4c5a-827a-3091f155e3fd

![hover](https://github.com/icssc/AntAlmanac/assets/78244965/f9816b9d-ddda-4c0f-80f4-eeac92428612)


## Test Plan
- Verify that the graph pops up in the Search and Added tabs.
- Verify that the graph looks okay on mobile devices.
- Verify that the "View on Zotistics" link still works.

## Issues

Closes #672
